mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -707,7 +716,7 @@ private ReactiveClassicGenericQueryExecutor getQueryExecutor(String tenantId) {
     return postgresClientFactory.getQueryExecutor(tenantId);
   }
 
-  private PgPool getCachecPool(String tenantId) {
+  private PgPool getCachePool(String tenantId) {
```

Comment:
Thanks for finding the typo. It should however be `getCachedPool`.

Reply:
@wwelling done